error.go

```diff
@@ -294,6 +294,23 @@ func (e *RateLimitError) Error() string {
 	return e.stripeErr.Error()
 }
 
+// redact returns a copy of the error object with sensitive fields replaced with
+// a placeholder value
```

Comment:
Super nit, but mind punctuating here (it's pretty common throughout the library now).